meja/tests/handler.meja

```diff
@@ -0,0 +1,14 @@
+request('a) A;
+
+handler A => {
```

Comment:
Is the assumption that each request has exactly one handler? I would expect something more like

```reasonml
let handle_A = handler {
  | A => unhandled
};
```
which would compile to
```ocaml

let handle_A = function
  | With {request= A; respond} ->
      let unhandled = Snarky.Request.unhandled in
      unhandled
  | _ ->
      Snarky.Request.unhandled
```

though then you have to "register them" separately.

Reply:
Done. I'm going to merge this into #273 so I can get it integrated with the stdlib.